src/LP.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.14;
+
+import "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import "lib/openzeppelin-contracts/contracts/utils/math/Math.sol";
+
+contract LiquidityPool {
+    using Math for uint256;
+
+    IERC20 public tokenA;
+    IERC20 public tokenB;
+    address public owner;
+    uint256 public totalLiquidity;
+    mapping(address => uint256) public liquidity;
+    uint256 public fee = 30; // 0.30% fee
+
```

Comment:
you could add another constant to represent 1 in bips i.e. 10000 (and not 1000 like you put in your code).

`uint256 public constant ONE_IN_BPS = 1e4;`